lazy_tensor_core/lazy_tensor_core/csrc/ts_backend/ts_computation_client.cpp

```diff
@@ -47,7 +46,9 @@ std::vector<ComputationClient::DataPtr> TSComputationClient::ExecuteComputation(
           torch_lazy_tensors::compiler::ts_backend::GenericComputationTS*>(
           computation.computation())
           ->graph();
-  torch::jit::GraphExecutor interp(graph, "");
+  if (interpreters.find(graph) == interpreters.end()) {
```

Comment:
@desertfire I wonder why we decided to not store `GraphExecutor` as a part of `GenericComputationTS` along with a jit graph? We seem to be introducing another level of caching (really not a big deal). 
1. We look up a computation object `LookupCachedCompile` that owns a JIT graph
2. In TSClient you added another look up for the executor
```
I wonder if we could just in TSClient check if a GraphExecutor exists in the computation object and if so reuse it, otherwise create it and store in the `GenericComputationTS` computation? I'm also wondering if maybe we could even skip storing a jit graph in  `GenericComputationTS` and just instantiate `GraphExecutor` when we create  `GenericComputationTS` in the first place?

Reply:
Good point. I wasn't sure if GenericComputationTS and graph is one-to-one mapping, but it looks like that is true. Let me fix it.